kvdb/sqlbase/db.go

```diff
@@ -0,0 +1,267 @@
+//go:build kvdb_postgres
+
+package sqlbase
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+	"io"
+	"sync"
+	"time"
+
+	"github.com/btcsuite/btcwallet/walletdb"
+)
+
+const (
+	// kvTableName is the name of the table that will contain all the kv
+	// pairs.
+	kvTableName = "kv"
+)
+
+// Config holds a set of configuration options of a sql database connection.
+type Config struct {
```

Comment:
Even though this commit is mostly a move-commit, it is not very easy to review. There is no good side-by-side diff that makes it's clear what the actual changes are. Is it possible to restructure so that it's clear at a glance what is moved and what is changed? Maybe some of the changes can be extracted in a separate commit first, so that only a pure move remains?

Reply:
fair enough - have updated to split this up a bit more

Reply:
That turned out nice :+1: